tests/pr_test.go

```diff
@@ -66,3 +69,23 @@ func TestRunUpgradeBasicExample(t *testing.T) {
 		assert.NotNil(t, output, "Expected some output")
 	}
 }
+
```

Comment:
should the basic example now move to other_test.go like we did for all ICD modules? I think all the features will now be covered by fscloud complete example